python/mxnet/_numpy_op_doc.py

```diff
@@ -677,3 +677,50 @@ def _np_trace(a, offset=0, axis1=0, axis2=1, out=None):
     (2, 3)
     """
     pass
+
+
+def moveaxis(a, source, destination):
+    """Move axes of an array to new positions.
+
+    Other axes remain in their original order.
+
+    Parameters
+    ----------
+    a : ndarray
+    The array whose axes should be reordered.
```

Comment:
Add spaces here.

Reply:
can you have find other problems , i will fix them together

Reply:
Sure, I'm reviewing this file.